tests/beaker_tests/cisco_bgp/test_bgp.rb

```diff
@@ -258,6 +258,20 @@ def unsupported_properties(tests, id)
   unprops
 end
 
+def version_unsupported_properties(_tests, _id)
+  unprops = {}
+  if platform[/n7k/]
+    unprops[:disable_policy_batching_ipv4] = '8.1.1'
```

Comment:
Added comments, examples to version_unsupported_properties method in utilitylib.rb file as well